### ISSUE TITLE
Bootstrap 5

### ISSUE THREAD

**Issue by jcdhan1** [thaider/Tweeki#262]
The site I need this skin for currently uses [FandomDesktop](https://community.fandom.com/wiki/Help:FandomDesktop). Statistics indicate that most users—both anonymous and logged-in—prefer to browse in the skin's built-in dark mode.

Tweeki would fit the site's extensions better as it relies on Semantic MediaWiki with page forms. 
Bootstrap 5.3.2 has a way of making dark modes. [The installation page](https://tweeki.kollabor.at/wiki/Installation) says version 3 which supports Bootstrap 5 was expected earliest end of 2022. Is there anything holding this release back?

[There are guides on how to migrate from Bootstrap 4](https://getbootstrap.com/docs/5.2/migration/).

**Comment by jcdhan1** [comment]
Hello @thaider 

I've made a fork of this skin to address the needs of a Wiki that has it (primarily, our users prefer a dark theme). I've checked and tested all the breaking changes in [the official list](https://getbootstrap.com/docs/5.3/migration). Would it be fine to create a pull request?

**Comment by thaider** [comment]
Hi. A pull request would be very welcome. I won't have time to look into it before mid of December, though.

**Comment by thaider** [comment]
Yes, you can find a working version in the 5.39 and development branches.
